pynecone/utils/format.py

```diff
@@ -167,6 +167,9 @@ def format_string(string: str) -> str:
     Returns:
         The formatted string.
     """
+    # Replace string interpolation from Python style to JS style
+    string = string.replace("{", "${")
```

Comment:
This currently won't allow us to have "{" in any text so we can't do something like

```python
 pc.text("Test {}")
```

I think we may need to use a regex and search for matches of `"{state.*}"` and adding the $ then. 

We need to use the `state.get_name()` method to get the name of the state, not sure what would be the easiest way to pass it down here.

Reply:
Hi, you're absolutely right, sorry i've forgot that case.
Anyway i think that i've fixed it, some tests are required.
I will post here the new code so you can review it asap.
Some changes are required i think.